parsing/lexer.mll

```diff
@@ -665,8 +730,10 @@ and comment = parse
         is_in_string := false;
         store_string_char '\"';
         comment lexbuf }
-  | "{" ('%' '%'? extattrident blank*)? (lowercase* as delim) "|"
-      {
+  | "{" ('%' '%'? extattrident blank*)? (ident_ext? as raw_delim) "|"
+      { match lax_delim raw_delim with
```

Comment:
It seems weird that delimited strings are not recognized in comments because of an encoding error in an arbitrary string. What would happen if invalid ident chars were allowed here as well as uppercases ?